pokete.py

```diff
@@ -1101,26 +1145,27 @@ def add(self):
 
 class Buy:
     """Menu to buy items in, is triggered in shop"""
+
     def __init__(self):
-        self.box = ChooseBox(height-3, 35, "Shop")
-        self.box2 = Box(7, 21,)
+        self.box = ChooseBox(height - 3, 35, "Shop")
+        self.box2 = Box(7, 21, )
         self.items = [Inv.poketeball, Inv.superball, Inv.healing_potion,
-                    Inv.super_potion, Inv.ap_potion]
+                      Inv.super_potion, Inv.ap_potion]
         self.box.add_c_obs([se.Text(f"{obj.pretty_name} : {obj.price}$")
-                                for obj in self.items])
-        self.money_label = se.Text(str(figure.get_money())+"$")
+                            for obj in self.items])
+        self.money_label = se.Text(str(figure.get_money()) + "$")
```

Comment:
```suggestion
        self.money_label = se.Text(f"{figure.get_money()}$")
```
An f-string may be suited better.